src/bosh-deployment/docker/cpi.yml

```diff
@@ -30,6 +30,7 @@
     - 25250/tcp # blobstore
     - 8443/tcp # uaa
     - 8080/tcp # cfg serv
+    - 443/tcp # concourse
```

Comment:
Coul you create an additional ops file for you changes, since the bosh-deployment dir gets overwritten by ci. You can see the layout here: https://github.com/starkandwayne/bucc/tree/master/ops/cpis/virtualbox

You probably also want:
- 8844: credhub
- 8200: vault credhub proxy (`bucc safe`)

To check if all services are accessible you can run `bucc test`